vm-setup/roles/v1a2_integration_test/tasks/verify.yml

```diff
@@ -0,0 +1,96 @@
+---
+  - name: Get the IP of controlplane node
+    shell: |
+      kubectl get bmh -n "{{ NAMESPACE }}" -o json | jq -r '.items[]
+      | select (.spec.consumerRef.name == "{{ CLUSTER_NAME }}-controlplane-0")
+      | .status.hardware.nics[] | select (.name == "eth1") | .ip'
+    register: controlplane_ip
+
+  - name: Define variable for controlplane node IP
+    set_fact:
+      bmh_ip: controlplane_ip.stdout
+
+  - name: Define username variable for ubuntu node
+    set_fact:
+      username: ubuntu
+    when: IMAGE_OS == "Ubuntu"
+
+  - name: Define username variable for target centos node
+    set_fact:
+      username: centos
+    when: IMAGE_OS == "Centos"
+
+  - name: Verification test
+    block:
+      - name: Waiting for controlplane to come up
+        wait_for:
+          port: 22
+          state: started
+          timeout: 1200
+          delay: 10
+
+      - name: Install jq
+        apt:
+          name: jq
+          state: present
+        when:
+          ansible_distribution == "Ubuntu"
+
+      - name: Install jq
+        yum:
+          name: jq
+          state: present
+        when:
+          ansible_distribution == "CentOS"
+
+      # Install Calico
+      - name: Download Calico manifest
+        get_url:
+          url: https://docs.projectcalico.org/v3.9/manifests/calico.yaml
+          dest: /tmp/
+        register: calico_manifest
+
+      - name: Replace the POD_CIDR in calico config
+        replace:
+          path: /tmp/calico.yaml
+          regexp: "192.168.0.0/16"
+          replace: '{{ baremetal_network }}'
+        register: updated_manifest
```

Comment:
no, this should be the pods CIDR, as defined in the cluster object.

Reply:
isn't it suppose to be `baremetal_network_cidr` as defined in common variables?